config/crd/bases/dws.cray.hpe.com_persistentstorageinstances.yaml

```diff
@@ -127,7 +198,7 @@ spec:
                 enum:
                 - creating
```

Comment:
Does `creating` go away?

Reply:
Only in the spec section. The spec state to active or destroying based on what the workflow controller wants. Then the PersistentStorageInstance controller sets the status state to creating, active, or destroying based on the actual state.